thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/DataCompletenessResource.java

```diff
@@ -1,52 +1,69 @@
 package com.linkedin.thirdeye.dashboard.resources;
 
+import com.linkedin.thirdeye.completeness.checker.DataCompletenessConstants;
+import com.linkedin.thirdeye.completeness.checker.DataCompletenessConstants.DataCompletenessAlgorithmName;
+import com.linkedin.thirdeye.completeness.checker.DataCompletenessUtil;
+import com.linkedin.thirdeye.completeness.checker.PercentCompletenessFunctionInput;
+import com.linkedin.thirdeye.datalayer.bao.DataCompletenessConfigManager;
+import com.linkedin.thirdeye.datalayer.dto.DataCompletenessConfigDTO;
+import java.util.ArrayList;
 import java.util.List;
-
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.MediaType;
-
+import javax.ws.rs.core.Response;
 import org.apache.commons.collections.CollectionUtils;
 
-import com.linkedin.thirdeye.completeness.checker.DataCompletenessConstants.DataCompletenessAlgorithmName;
-import com.linkedin.thirdeye.completeness.checker.PercentCompletenessFunctionInput;
 
 @Path(value = "/data-completeness")
 @Produces(MediaType.APPLICATION_JSON)
 public class DataCompletenessResource {
 
+  private final DataCompletenessConfigManager dataCompletenessManager;
+
+  public DataCompletenessResource(DataCompletenessConfigManager dataCompletenessManager) {
+    this.dataCompletenessManager = dataCompletenessManager;
+  }
```

Comment:
I believe we follow the convention of naming the Manager instances as dataCompletenessDAO. Would be good to continue that